README.md

```diff
@@ -59,14 +59,9 @@ and health-checking mechanisms.
   sudo yum -y install consul-1.9.1
   ```
 
-* Install Motr.
+* Build and Install Motr.
 
-  * .. from RPMs
-    ```sh
-    sudo yum -y install cortx-motr cortx-motr-devel
-    ```
-
-  * .. or from [sources](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) use this link to build Motr from source. Once you built Motr continue with the steps listed below.
+  *  from sources use [this](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) link to build Motr from source. Once you built Motr continue with the steps listed below.
```

Comment:
Follow [Motr quick start guide](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) to build Motr from source. After compiling Motr sources, please continue with the below steps to build Hare using Motr sources.

Reply:
Resolved these comments as part of #1610

Reply:
why a new as part of #1610 (a new PR) and why not the same one?

Reply:
by mistake created another PR